packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/FEaaSWrapper.tsx

```diff
@@ -0,0 +1,19 @@
+import { FEaaSComponent } from '@sitecore-jss/sitecore-jss-nextjs';
+import React from 'react';
+
+type FEaaSWrapperProps = {
+  params: { [key: string]: string };
+};
+
+export const Default = (props: FEaaSWrapperProps): JSX.Element => {
```

Comment:
Just one suggestion, may be you could reuse `FEaaSComponentProps`? (can be imported from sitecore-jss-nextjs)